docs/userguides/quickstart.md

```diff
@@ -96,6 +96,45 @@ sudo rm -rf **\~/.solcx**
 sudo rm -rf **\~/.vvm**
 ```
 
+## Overview
+
+The ApeWorx project folder hierarchy:
+
+```
+project              # The root project directory
+├── contracts/       # Project source files, such as '.sol' or '.vy' files
+├── tests/           # Project tests, ran using the 'ape test' command
+├── scripts/         # Project scripts, such as deploy scripts, ran using the 'ape run <name>' command
+└── ape-config.yaml  # The ape project configuration file
+```
+
+## Enviroment Variables:
+
+Enviroment Variables are used to help connect you to your files or ecosytems outside of ApeWorx
+
+Please setup environment variables (where applicable) and follow the latest instructions from the 3rd party:
+
+Example of a use case:
+
+```bash
+export WEB3_INFURA_PROJECT_ID={YOUR_INFURA_PROJECT_ID}
+
+export WEB3_ALCHEMY_API_KEY={YOUR_ALCHEMY_KEY}
```

Comment:
Typically I use `<something goes here>` because the arrows are not valid shell commands
```suggestion
# Used by the `ape-infura` plugin
export WEB3_INFURA_PROJECT_ID=<YOUR_INFURA_PROJECT_ID>

# Used by the `ape-alchemy` plugin
export WEB3_ALCHEMY_API_KEY=<YOUR_ALCHEMY_KEY>
```